Cargo.toml

```diff
@@ -18,7 +18,9 @@ tokio = { version = "1.21", optional = true, default_features = false, features
 serde_json = "1.0.64"
 
 [features]
-# Adds I/O safety traits introduced in Rust 1.63
+# Adds `fs_err_try_exists` to `Path`, introduced in Rust 1.6.3
+path_try_exists = []
+# Adds I/O safety traits, introduced in Rust 1.63
```

Comment:
I'm not sure best practice on features. Other things I considered include:

Adding an `all` feature that loads all features:


```
all = ["path_try_exists", "io_safety"]
```

Alternatively making a feature named `1_63` that would enable all features on 1.63+.

```
1_63 = ["path_try_exists", "io_safety"]
```